week-1/mandatory/2-classes-db/task.md

```diff
@@ -18,19 +18,39 @@ When you have finished all of the questions - open a pull request with your answ
 If you haven't completed all the exercises from this lesson then do that first.
 
 ### Tasks
+
 1.  Which rooms have a rate of more than 100.00?
+    select room_no, rate from rooms where rate > 100.00;
 2.  List the reservations that have a checkin date this month and are for more than three nights.
+    select \*, checkin_date - checkout_date as night from reservations where checkin_date between '2020-09-01' and '2020-09-30' and checkout_date - checkin_date > 3
 3.  List all customers from cities that begin with the letter 'M'.
+    select \* from customers where name like 'M%';
 
 Insert some new data into the room_types and rooms tables, querying after each stage to check the data, as follows:
 
 4.  Make a new room type of PENTHOUSE with a default rate of £185.00
+    INSERT INTO room_types (room_type, def_rate) VALUES('PENTHOUSE',185.00);
 5.  Add new rooms, 501 and 502 as room type PENTHOUSE and set the room rate of each to the default value (as in the new room type).
+    INSERT INTO rooms (room_no, room_type, rate) VALUES (501, 'PENTHOUSE', 185), (502,'PENTHOUSE', 185);
 6.  Add a new room 503 as a PREMIER PLUS type similar to the other PREMIER PLUS rooms in the hotel but with a room rate of 143.00 to reflect its improved views over the city.
+    INSERT INTO ROOMS (room_no, room_type, rate) VALUES (503,'PREMIER PLUS', 143.00);
 
 Using what you can learn about aggregate functions in the w3schools SQL classes (or other providers), try:
 
 7.  The hotel manager wishes to know how many rooms were occupied any time during the previous month - find that information.
+    SELECT count(cust_id) from reservations where checkin_date >= '2020-08-01' and checkout_date < '2020-08-31';
```

Comment:
the one makes sense is to count room_no, using (*) or (1) will do same trick, all columns will work in your case, but it doesn't look easy to understand.
Also checkout date is added